templates/cube/study/_navigation.md

```diff
@@ -0,0 +1,75 @@
+### Cube Study Materials
+- [Index](/cube/study)
+
+### Ubuntu System Architecture
+- [Unit 1](/cube/study/system-architecture/unit-1)
```

Comment:
Seems I don't have permission to edit this file directly.

Could you please either:

 * Change "Unit" to "Part" in the chapter names in the left navbar.
 * Grant me editing permissions so that I can make the change myself.

Thank you!